forge-gui/src/main/java/forge/gui/card/CardDetailUtil.java

```diff
@@ -296,9 +296,7 @@ public static String composeCardText(final CardStateView state, final GameView g
             area.append("Effect");
         }
         // card text
-        if (area.length() != 0) {
-            area.append("\n");
-        }
+        area.append("\n");
```

Comment:
![image](https://github.com/user-attachments/assets/7fef92ce-2d73-418c-b600-f0d87a18d113)

Reply:
I think the idea was to clean that up by removing the conditionals and just trim off the blank line at the end. Then I guess I forgot to do the second part of that. Should look right now.